packages/thrift-server-core/src/main/types.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  TProtocol,
+  TProtocolConstructor,
+  TTransportConstructor,
+} from 'thrift'
+
+export type ProtocolType =
+  'binary' | 'compact' | 'json'
+
+export type TransportType =
+  'buffered' | 'framed'
+
+export interface IPluginOptions {
```

Comment:
This should live in `thrift-server-hapi`

Reply:
It's also used by express